maps/outreach/outreach_overmap.dm

```diff
@@ -0,0 +1,50 @@
+// These defines are moved here, as we don't want this to generate or require these paths when testing other maps.
+/datum/map/outreach
+	overmap_ids = list(OVERMAP_ID_SPACE = /datum/overmap/outreach)
+
+/obj/effect/overmap/star/outreach
+	name = "Castor"
+
+/////////////////////////////////////////////////////////////////////////////////
+// Overmap Details
+/////////////////////////////////////////////////////////////////////////////////
+/datum/overmap/outreach
+	event_areas = 10
+	map_size_x  = 50
+	map_size_y  = 50
+	var/map_file// = "maps/outreach/outreach-overmap.dmm"
```

Comment:
Should this be commented out?

Reply:
Oh, that actually reminds me I didn't test the overmap -_-
But yeah that's fine. We really shouldn't need to pre-generate the overmap.